host/discoverd.go

```diff
@@ -105,3 +109,15 @@ func (d *DiscoverdManager) ConnectPeer(ips []string) error {
 	}
 	return err
 }
+
+func (d *DiscoverdManager) UpdateTags(tags map[string]string) error {
+	d.mtx.Lock()
+	defer d.mtx.Unlock()
+	for k, v := range tags {
+		d.inst.Meta[host.TagPrefix+k] = v
```

Comment:
> $ flynn-host tags set foo=     # foo gets deleted

This would surprise me. Not really clear for the user it would drop a tag. Even more when the single absence of `=` would cause it to get created instead:

> $ flynn-host tags set foo      # foo gets set to "true"

Aren't the `-` prefixing a tag more clearer?